rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf

```diff
@@ -558,21 +558,31 @@ SecRule ARGS_NAMES|ARGS|XML:/* "@rx (?i)(?:not\s+between\s+(?:(?:(?:'[^']*')|(?:
 #
 # -=[ SQL Tautologies ]=-
 #
-# Regexp generated from util/regexp-assemble/data/942130.data using Regexp::Assemble.
+# Boolean-based SQL injection or tautology attack. Boolean values (True or False) are used to carry out
+# this type of SQL injection. The malicious SQL query forces the web application to return a different result de-
+# pending on whether the query returns a TRUE or FALSE result.
+#
+# The original 942130 was split in two rules:
+# - 942130 targeting "true" tautologies (e.g. 1 = 1)
+# - 942131 targeting "false" tautologies (e.g. 1 != 2)
+#
+# The regexp written at the initial rule on the chain is the original prefix from the 942130.data.
+# When the prefix matches, we store the match to a new variable for comparing in the chained rule.
+#
+# The chained rule then uses the assembled regexp from 942130.data to check for the right hand side of the logical query.
+#
 # To rebuild the regexp:
 #   cd util/regexp-assemble
-#   ./regexp-assemble.py 942130
+#   ./regexp-assemble.py data/942130.data
```

Comment:
This comment needs to be updated. The regular expression for the starter rule can't be generated (at the moment).